src/components/Editor/index.js

```diff
@@ -188,11 +197,14 @@ class Editor extends Component {
   }
 
   render() {
-    const { contentEditable, className, style } = this.props
+    const { contentEditable, className, style, ...rest } = this.props
     const { html } = this.state
+    delete rest.children;
```

Comment:
Unfortunately, setting it to `undefined` does not have the desired effect.  It's still a property on `rest` and causes a React warning: Unknown prop `code` on <pre> tag.  I removed the `delete` on children and the unneeded semicolon.  The only other option is destructuring out `code` and ignoring it, but I don't like unused parameter warnings much.